docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,336 @@
+
+In case of the **PARALLEL** interface:
+
+.. code-block::
+
+   make SER_PAR_N=0
+
+In case of the **SERIAL** interface:
+
+.. code-block::
+
+   make SER_PAR_N=1
+   
+.. note::
+
+   This switch is a 'hardware' switch. Please rebuild the  design if the 
+   variable has been changed.
+   
+   -   SL5 - unmounted - Parallel interface
+   -   SL5 - mounted - Serial interface
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ========= ====================================
+Jumper/Solder link Position  Description
+================== ========= ====================================
+SL1                Unmounted Channel Sequencer Enable
+SL2                Unmounted RC Enable Input
+SL3                Mounted   Selects 2 MISO mode
+SL4                Unmounted Oversampling Ratio Selection OS2
+SL5                Mounted   If mounted, selects serial interface
+SL6                Unmounted Oversampling Ratio Selection OS1
+SL7                Unmounted Oversampling Ratio Selection OS0
+LK40               A         Onboard 5v0 power supply selected
+LK41               A         Onboard 3v3 power supply selected
+================== ========= ====================================   
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+AD7616_SDZ serial interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ../images/ad7616_sdz/ad7616_serial_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD7616_SDZ using the serial interface block diagram
+   
+AD7616_SDZ parallell interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ../images/ad7616_sdz/ad7616_parallel_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD7616_SDZ using the parallel interface block diagram   
+
+IP list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`AD_EDGE_DETECT <master:library/common/ad_edge_detect.v>`
+-  :git-hdl:`AXI_AD7616 <master:library/axi_ad7616>` *
+-  :git-hdl:`AXI_CLKGEN <master:library/axi_clkgen>`
+-  :git-hdl:`AXI_DMAC <master:library/axi_dmac>`
+-  :git-hdl:`AXI_HDMI_TX <master:library/axi_hdmi_tx>`
+-  :git-hdl:`AXI_I2S_ADI <master:library/axi_i2s_adi>`
+-  :git-hdl:`AXI_PWM_GEN <master:library/axi_pwm_gen>`
+-  :git-hdl:`AXI_SPDIF_TX <master:library/axi_spdif_tx>`
+-  :git-hdl:`AXI_SPI_ENGINE <master:library/spi_engine/axi_spi_engine>` **
+-  :git-hdl:`AXI_SYSID <master:library/axi_sysid>`
+-  :git-hdl:`SPI_ENGINE_EXECUTION <master:library/spi_engine/spi_engine_execution>` **
+-  :git-hdl:`SPI_ENGINE_INTERCONNECT <master:library/spi_engine/spi_engine_interconnect>` **
+-  :git-hdl:`SPI_ENGINE_OFFLOAD <master:library/spi_engine/spi_engine_offload>` **
+-  :git-hdl:`SYNC_BITS <master:library/util_cdc/sync_bits.v>`
+-  :git-hdl:`SYSID_ROM <master:library/sysid_rom>`
+
+.. note::
```

Comment:
Here we can do 
```
.. admonition:: Legend
   :class: note
     
   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
```
To have "Legend" as the note title.

Same for the other 3 "Legend" instances.

Reply:
Thank you.

Reply:
Done.